plugins/DuskVerb/tools/tuner/eq_match_curve.py

```
@@ -0,0 +1,151 @@
+        if stem == "auto":
+            sys.exit("no matching stimulus pair found in both dirs")
+    dv = find_stim(args.dv_dir, stem)
+    lx = find_stim(args.lex_dir, stem)
+    if not dv or not lx:
+        sys.exit(f"missing {stem} render in dv_dir or lex_dir")
+
+    # For non-sustained stimuli, the steady-state window is meaningless;
+    # use post-peak windows instead.
+    if stem != "sustained":
+        # Reset to post-peak windows: use peak-aligned [50ms, 1.0s].
+        x_dv, sr_dv = sf.read(dv); m_dv = x_dv.mean(axis=1) if x_dv.ndim>1 else x_dv
```

Ruff (0.15.15)

[error] Multiple statements on one line (semicolon)

(E702)